src/configfile.c

```diff
@@ -185,6 +186,7 @@ static int config_insert(server *srv) {
 	cv[72].destination = &(srv->srvconf.http_header_strict);
 	cv[73].destination = &(srv->srvconf.http_host_strict);
 	cv[74].destination = &(srv->srvconf.http_host_normalize);
+	cv[75].destination = &(s->use_acceptfilter);
```

Comment:
Please move this line inside the loop immediately below, as it is T_CONFIG_SCOPE_CONNECTION.  (Sorry, the config option parsing code is more complex than I'd like.)